thriftpy/contrib/tracking/__init__.py

```diff
@@ -123,13 +136,20 @@ def _req(self, _api, *args, **kwargs):
             )
             self.tracker.record(header_info, exception)
 
+    def check_support(self, version):
+        return hasattr(self, 'server_version') \
+               and self.server_version is not None \
+               and self.server_version >= version \
+               and self.client_version >= version
+
 
 class TTrackedProcessor(TProcessor):
     def __init__(self, tracker_handler, *args, **kwargs):
         super(TTrackedProcessor, self).__init__(*args, **kwargs)
-
         self.tracker = tracker_handler
         self._upgraded = False
+        self.add_response_header = False
+        self.server_version = TrackerVersion.support_response_header
```

Comment:
self.add_response_header = False
self.server_version = TrackerVersion.support_response_header

Maybe... the second should be constant
The first should not.